lib/mapper.js

```diff
@@ -91,7 +97,7 @@ module.exports = function mapper (routePlan) {
   return routes
 }
 
-function buildPath (route) {
+function buildPath(route) {
```

Comment:
Can you fix this (and the other linting errors?)

Reply:
@tswaters I think my pull request is no longer necessary since you have highlighted that passing instance in the routes does not in cross communication